scripts/common/sonic-wpa-supplicant-build/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -ex
+
+# Install HIREDIS
+sudo apt-get install -y libhiredis0.14 libhiredis-dev
+
+# Install REDIS
+sudo apt-get install -y redis-server
+sudo sed -ri 's/^# unixsocket/unixsocket/' /etc/redis/redis.conf
+sudo sed -ri 's/^unixsocketperm .../unixsocketperm 777/' /etc/redis/redis.conf
+sudo sed -ri 's/redis-server.sock/redis.sock/' /etc/redis/redis.conf
+sudo service redis-server start
```

Comment:
Can it be run in the docker container? One agent may run several test builds in parallel.

Reply:
agree, we should try to run this build in sonic-slave docker.

Reply:
btw, @xumia, i think the build is running inside sonic-slave-buster.